bin/lang-js/src/function_kinds/resolver_function.ts

```diff
@@ -62,29 +63,39 @@ export interface TypeCheckSuccess {
 
 export type TypeCheckResult = TypeCheckFailure | TypeCheckSuccess;
 
-const isArray = (value: unknown): TypeCheckResult => (_.isArray(value)
+const isArray = (
+  value: unknown,
+): TypeCheckResult => (_.isArray(value)
   ? { valid: true }
   : { valid: false, message: "Return type must be an array." });
 
-const isBoolean = (value: unknown): TypeCheckResult => (_.isBoolean(value)
+const isBoolean = (
+  value: unknown,
+): TypeCheckResult => (_.isBoolean(value)
   ? { valid: true }
   : { valid: false, message: "Return type must be a boolean." });
 
-const isInteger = (value: unknown): TypeCheckResult => (_.isInteger(value)
+const isInteger = (
```

Comment:
it doesn't look like these 3 helper functions are used? below it just uses the lodash fns

Reply:
If I understand correctly, they are used by`typeChecks`  [here](https://github.com/systeminit/si/blob/main/bin/lang-js/src/function_kinds/resolver_function.ts#L223). I didn't actually change this bit, just formatting.